app/presenters/matrix_box_presenter.rb

```diff
@@ -44,13 +43,17 @@ def rss_log_to_presenter(rss_log, view)
       end
     self.where = view.location_link(target.place_name, target.location) \
                  if target&.respond_to?(:location)
+    self.time = rss_log.updated_at
 
     self.thumbnail =
       if target&.respond_to?(:thumb_image) && target&.thumb_image
         view.thumbnail(target.thumb_image,
                        link: { controller: target.show_controller,
                                action: target.show_action, id: target.id })
       end
+    return unless rss_log.detail
+
+    self.detail = rss_log.detail
```

Comment:
Can we do this without calling rss_log.detail twice?  Maybe something like this:

    return unless temp = rss_log.detail
    self.detail = temp

Reply:
Just committed this change.